samples/ipc_call_interface/inc/math.h

```diff
@@ -2,7 +2,51 @@
 // Licensed under the MIT license.
 // See LICENSE file in the project root for full license information.
 
+#include "az_ulib_ipc_api.h"
 #include "az_ulib_result.h"
+#include <stdint.h>
 
+/*
+ * interface definition
+ */
+#define MATH_INTERFACE_NAME "math"
```

Comment:
Do you think that it would be better if this file along with the src file live in a separate "interface" directory instead of in the same directory as the files that call it? I think it would be good to separate the calling files from the publishing files to show the modularity of this design.

Reply:
Sounds good to me.

---
In reply to: [529828574](https://github.com/Azure/azure-ulib-c/pull/53#discussion_r529828574) [](ancestors = 529828574)